config.rb

```diff
@@ -96,7 +96,7 @@ def self.test?
   optional :vm_pool_project_id, string
 
   # Minio
-  override :minio_host_name, "storage.ubicloud.com", string
+  override :minio_host_name, "minio.ubicloud.com", string
```

Comment:
I think the host name depends on the decision we will make about selling minio clusters vs selling blob storage. For now either is OK I guess. We just need to be careful about not depending this host name in a way that is difficult to reverse.